### PR TITLE
feat: Add social media bulk tweet generator

### PR DESCRIPTION
**Pull request by iamareebjamal**
image

### CODE REVIEW

File: scripts/generate_session_tweet_csv.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+template = input(
+    f'Default Template: {default_template}\nPress enter to use default, or type to override: '
+)
+
+template = template.strip()
+
+print(f'Using template: { template or default_template }')
+
+duration_string = input(
+    'How much time ago tweet should be scheduled?\n[e.g. 4 hours, 5 minutes, 2 hours 30 minutes, etc]\n'
```
Comment (mariobehling) [inline scripts/generate_session_tweet_csv.py:35]: How much time before the session should the post be scheduled?

**Comment by mariobehling** [comment]
Is it possible to add the Twitter handle of the speakers too?

Nice script! Lets schedule a meeting with Hong Phuc to test this in the upcoming days.

**Comment by iamareebjamal** [comment]
Twitter handle is added if it is present

**Review by mariobehling — APPROVED** [review]
Please change "How much time ago tweet should be scheduled?\n[e.g. 4 hours, 5 minutes, 2 hours 30 minutes, etc]" -> When should the session post on social media be scheduled before the actual session time? For example, 4 hours, 5 minutes, 2 hours 30 minutes, etc."

File: scripts/generate_session_tweet_csv.py (hunk 1 above)
Comment (iamareebjamal) [inline scripts/generate_session_tweet_csv.py:35]:
```suggestion
    'When should the session post on social media be scheduled before the actual session time?\nFor example, 4 hours, 5 minutes, 2 hours 30 minutes, etc.\n'
```

**Pull request merged**